sdk/python/kfp/compiler/compiler.py

```diff
@@ -30,15 +30,17 @@
 from .. import dsl
 from ._k8s_helper import convert_k8s_obj_to_json, sanitize_k8s_name
 from ._op_to_template import _op_to_template, _process_obj
-from ._default_transformers import add_pod_env
+from ._default_transformers import add_pod_env, add_pod_labels
 
 from ..components.structures import InputSpec
 from ..components._yaml_utils import dump_yaml
 from ..dsl._metadata import _extract_pipeline_metadata
 from ..dsl._ops_group import OpsGroup
 from ..dsl._pipeline_param import extract_pipelineparams_from_any, PipelineParam
 
-
+_SDK_VERSION_LABEL = 'pipelines.kubeflow.org/kfp_sdk_version'
```

Comment:
Since this is new, can we make the name more consistent and use dashes, e.g.

```
_SDK_VERSION_LABEL = 'pipelines.kubeflow.org/kfp-sdk-version'
```

Reply:
I followed the label name we used for pipeline https://github.com/kubeflow/pipelines/blob/6b2fc775291a3eaaa6643b8a33c230d13c2f6c06/sdk/python/kfp/compiler/compiler.py#L960

Reply:
Looks like all labels use underscore instead of dash

Reply:
'pipelines.kubeflow.org/pipeline-sdk-type' is already used in TFX therefore we can't change the dash to underscore

Reply:
Thanks Xiyue. Let's go with your approach.